src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -280,27 +281,30 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
   @Watch('getShowErrors')
   @Watch('minAuthorizationDate') // because Authorization Date depends on this
   @Watch('authorization.date') // because Expiry Date depends on this
+  @Watch('getExistingBusinessInfo.homeIncorporationDate') // because Authorization Date validity depends on this
```

Comment:
In Extrapro situation, if the user select a home incorporation date later than BC registry date:
the min authorization date will be BC registry date.
But the auth. date rule will still check if the authorization date >=  home incorporation date

Reply:
Are you sure you need this? I think that maybe line 282 will trigger this event already.

(Line 282 watches a computed variable, which _should_ auto-update if homeIncorporationDate changes.)

Reply:
> Are you sure you need this? I think that maybe line 282 will trigger this event already.
> 
> (Line 282 watches a computed variable, which _should_ auto-update if homeIncorporationDate changes.)

When the min authorization date is set to be bcFoundingDate (line 182). In that case, the change of homeIncorporateDate will not change the min authorization date. 
But the rule (line 199) still checks whether the authorization date >= home incorporation date.

Reply:
current screenshot: when homeIncorporateDate changes, the min authorization date does not change
and there will be an error
![image](https://github.com/bcgov/business-create-ui/assets/122495122/c481ccd3-6a9a-4ace-bf57-b37d9f38f3c3)
![image](https://github.com/bcgov/business-create-ui/assets/122495122/3a3d2ea6-3c6c-4b57-9f40-2146378bf3ab)

Reply:
> When the min authorization date is set to be bcFoundingDate (line 182). In that case, the change of homeIncorporateDate will not change the min authorization date. But the rule (line 199) still checks whether the authorization date >= home incorporation date.

I tested this locally. The watcher on line 282 does indeed fire, but the condition on line 286 is false, so the components are not validated. I think this is actually what we want: we do not show errors until the review step has been displayed (and getShowErrors is True).

Reply:
If I remove line 284, even when getShowErrors is True - if we change the date of incorporation, the error will not show
![image](https://github.com/bcgov/business-create-ui/assets/122495122/1db26d86-ec94-4546-87b2-38d4bde3ef16)
the error will only show when we try to change the authorization date again:
![image](https://github.com/bcgov/business-create-ui/assets/122495122/e0e5f839-eb0d-4bf0-b793-53b68968077e)

I tested with:
http://localhost:8080/continuation-in-business-home?id=ThevF8a4zF&accountid=3302
I changed the date of incorporation to a date later than authorization date.
It is because when minAuthorizationDate is set to be the bcFoundingDate (line 182). In that case, the change of homeIncorporateDate will not change the min authorization date, so line 282 didn't fire.

Reply:
In your example, the minimum authorization date is 1968-09-10. This comes from the expro record.

Test again but using a manually entered business.

Reply:
You can debug and step through the code to see what the variable values are, or do something like this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/a88be025-3c06-4ffe-93e1-1ba689d22647)

Reply:
> In your example, the minimum authorization date is 1968-09-10. This comes from the expro record.
> 
> Test again but using a manually entered business.

Yes, the minimum authorization date (1968-09-10) is the bcFoundingDate (line 182), and irrelevant to the homeIncorporateDate. 
So if we only watch the @Watch('minAuthorizationDate') (line 282), when the user changes the homeIncorporateDate (say to June 11 2024), the minAuthorizationDate won't change, so line 285 - 294 didn't run. The current authorization date is actually invalid (Jun 3 2024), but the error will not show, and the tick mark incorrectly shows.

Reply:
OK, Arwen and I sorted this out. (Needed to add another rule.) Looks good to me now. This conversation can be resolved.